config/test.yaml

```diff
@@ -0,0 +1,5 @@
+postgres:
+  user: postgres
+  password: postgres
+  dbname: iterable_test
```

Comment:
So we also need our migrations to run on db_test? @nare469 do you know how to do this? can we have db/migrate_test